examples/toolkits/browser_toolkit_with_cookie.py

```diff
@@ -0,0 +1,168 @@
+# ========= Copyright 2023-2024 @ CAMEL-AI.org. All Rights Reserved. =========
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ========= Copyright 2023-2024 @ CAMEL-AI.org. All Rights Reserved. =========
+
+import asyncio
+import gc
+import os
+import sys
+import time
+
+from playwright.sync_api import sync_playwright
+
+from camel.agents import ChatAgent
+from camel.configs import ChatGPTConfig
+from camel.models import ModelFactory
+from camel.toolkits import BrowserToolkit
+from camel.types import ModelPlatformType, ModelType
+
+
+def save_auth_cookie(cookie_json_path: str, url: str):
```

Comment:
Should we move this to utils?
Instead of waitiing for 60 second. Can we save on close?